book/07-git-tools/sections/rewriting-history.asc

```diff
@@ -284,7 +291,7 @@ Ref 'refs/heads/master' was rewritten
 
 The `--tree-filter` option runs the specified command after each checkout of the project and then recommits the results.
 In this case, you remove a file called `passwords.txt` from every snapshot, whether it exists or not.
-If you want to remove all accidentally committed editor backup files, you can run something like `git filter-branch --tree-filter 'rm -f *~' HEAD`.
+If you want to remove all accidentally committed editor backup files, you can run something like `git filter-branch --tree-filter 'rm -rf *~' HEAD`.
```

Comment:
The `-r` flag for `rm` just means "remove directories, too." It doesn't make sense in this context.

Reply:
Done.